admin/syslog-ng/Makefile

```diff
@@ -32,61 +39,70 @@ endef
 define Package/syslog-ng/conffiles
 /etc/syslog-ng.conf
 /etc/syslog-ng.d/
+/etc/scl.conf
 endef
 
 define Build/Configure
 	$(SED) 's,-I/usr/include,,' $(PKG_BUILD_DIR)/configure
 	$(Build/Configure/Default)
 endef
 
-CONFIGURE_ARGS += \
-  $(call autoconf_bool,CONFIG_IPV6,ipv6) \
-         --disable-dependency-tracking \
-         --disable-amqp \
-         --disable-tcp-wrapper \
-         --disable-glibtest \
-         --disable-mongodb \
-         --disable-java \
-         --disable-json \
-         --disable-python \
-         --disable-spoof-source \
-         --disable-sql \
-         --disable-linux-caps \
-	 --disable-smtp \
-	 --disable-redis \
-         --enable-prce \
-
-TARGET_CPPFLAGS += \
-  -I$(STAGING_DIR)/usr/include/eventlog
+CONFIGURE_ARGS +=  \
+	$(call autoconf_bool,CONFIG_IPV6,ipv6) \
+	--disable-dependency-tracking \
+	--enable-tcp-wrapper \
+	--disable-glibtest \
+	--disable-spoof-source \
+	--disable-sql \
+	--disable-java \
+	--disable-java-modules \
+	--disable-redis \
+	--disable-sql \
+	--disable-python \
+	--disable-smtp \
+	--enable-prce \
+	--disable-linux-caps \
+	--with-librabbitmq-client=no \
+	--with-mongoc=no \
+	--with-jsonc=system \
+	--enable-json=yes \
+	--enable-http=yes
 
 CONFIGURE_VARS += \
-  LIBDBI_CFLAGS="-I$(STAGING_DIR)/usr/include"
+	LIBDBI_CFLAGS="-I$(STAGING_DIR)/usr/include"
 
 define Package/syslog-ng/install
-	$(INSTALL_DIR) $(1)/usr/lib
-	$(MAKE) -C $(PKG_BUILD_DIR) \
-		install-sbinPROGRAMS install-libLTLIBRARIES \
-		install-moduleLTLIBRARIES DESTDIR="$(1)"
+	cd $(PKG_BUILD_DIR); make DESTDIR=$(1) install
+
+	$(call libtool_remove_files,$(1)) # This removes .la files in folder (including subfolders) /usr/lib
+	rm -rf $(1)/usr/lib/pkgconfig \
+	$(1)/usr/lib/*.a \
```

Comment:
```--enable-mixed-linking``` seems to be the cause having a quick look at the configure script, perhaps disable it?